Assets/Code/src/Runtime/Match/Components/MatchPauseController.cs

```diff
@@ -6,7 +6,7 @@ namespace HouraiTeahouse.FantasyCrescendo {
 
 public class MatchPauseController : MonoBehaviour {
 
-  public MatchManager MatchManager;
+  public MatchManager MatchManager { get { return MatchManager.Instance; } }
```

Comment:
Style: Please use expression bodied statement: `MatchManager =>MatchManager.Instance` for get-only one-liner properties.